client/src/+assets/landing.tsx

```diff
@@ -78,12 +78,12 @@ export const Landing: NextPage = () => {
                 Connect your wallet (Metamask) and purchase the Subscription you are interested in
               </UiText>
               <UiText type="p" variants={['detail']}>
-                Identify your desired product by using Nevermined's filter functions to select the
+                Identify your desired product by using Nevermined&lsquo;s filter functions to select the
```

Comment:
Maybe we shouldn't escape special characters? 🥺

Reply:
linter says that I should escape it